language/tools/move-cli/src/base/movey_login.rs

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) The Diem Core Contributors
```

Comment:
A nit. For new files, use only the "Move contributors" part of the copyright notice.